src/coreclr/vm/datadescriptor/datadescriptor.h

```diff
@@ -22,6 +22,9 @@
 
 #include "../debug/ee/debugger.h"
 #include "patchpointinfo.h"
+#ifdef FEATURE_COMWRAPPERS
+#include "../interop/comwrappers.hpp"
+#endif // FEATURE_COMWRAPPERS
```

Comment:
Can be removed once the definitions in `comwrappers.cpp` are moved to `interoplibabi.h`.

```suggestion
```